docs/getting-started/user-disclosure.md

```diff
@@ -0,0 +1,5 @@
+# Project-Provided User Disclosure
+
+Please allow the zwave-js project to collect some anonymized data regarding the devices you own so that we can generate statistics that allow us to better focus our development efforts. This information is not tracked to any identifiable user or IP address. Specifically, we'd like to collect: (1) a hash of your network ID to prevent duplicate records (this hash cannot be undone to reveal your network ID); (2) information about which version of zwave-js you are running; and (3) the manufacturer ID, product type, product ID, and firmware version of each device that is connected to zwave-js.
```

Comment:
```suggestion
Please allow the Z-Wave JS project to collect some anonymized data regarding the devices you own so that we can generate statistics that allow us to better focus our development efforts. This information is not tracked to any identifiable user or IP address. Specifically, we'd like to collect: (1) a hash of your network ID to prevent duplicate records (this hash cannot be undone to reveal your network ID); (2) name and version of the application you are running; (3) information about which version of `node-zwave-js` you are running; and (4) the manufacturer ID, product type, product ID, and firmware version of each device that is part of your Z-Wave network.
```